apps/shelver/mix.exs

```diff
@@ -36,6 +36,7 @@ defmodule Shelver.MixProject do
   # Type `mix help deps` for examples and options.
   defp deps do
     [
+      {:reflex, git: "https://bitbucket.org/disalazarg/reflex.git"},
```

Comment:
are you using a library that you created?

Reply:
@sespinoza-dev yup, available publicly albeit licenseless as sort of a pre-release alpha, this project serving as its first test in the real world; I'm planning to refactor it to integrate more cleanly with Absinthe/GraphQL, give it a F/OSS license and publish it on hex afterwards. Hopefully, before next year.